source/code/troubleshooter/modules/install/check_files.py

```diff
@@ -197,6 +197,9 @@ def check_dirs(dirs, exist_err, perms_err):
 def check_files(files, exist_err, perms_err):
     success = NO_ERROR
     for f in (files.keys()):
+        # skip over omsadmin.conf, as it's created in onboarding
+        if (f.endswith("omsadmin.conf")):
+            continue
```

Comment:
maybe print an info message? may be informative to know if it is missing; e.g. install and onboarding reported success but later the file is deleted